v2/samples/azureiotedge-simulated-temperature-sensor/Docker/linux-arm/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM microsoft/dotnet:2.0.0-runtime-stretch-arm32v7
```

Comment:
If we want this to run as an unprivileged user, you can pull from: azureiotedge/azureiotedge-module-base:1.0.0-preview001-linux-arm32v7, then after the COPY command, add:

USER moduleuser